packages/backend/src/services/authentication/authentication.service.ts

```diff
@@ -1,25 +1,27 @@
-import { Params, ServiceAddons, HooksObject } from '@feathersjs/feathers';
 import { AuthenticationService, JWTStrategy } from '@feathersjs/authentication';
 import { expressOauth, OAuthProfile, OAuthStrategy } from '@feathersjs/authentication-oauth';
+import { HooksObject, Params, ServiceAddons } from '@feathersjs/feathers';
 
 import { Application } from '../../declarations';
 import hooks from './authentication.hooks';
 
 declare module '../../declarations' {
   interface ServiceTypes {
-    authentication: AuthenticationService & ServiceAddons<any>;
+    authentication: AuthenticationService & ServiceAddons<unknown>;
```

Comment:
what is the type 'unkown' and where is the difference to any?

Reply:
`unknown` explicitly says that this variable and all of its properties are unknown to us. We simply don't know what is inside. `any` instead means "go for it" even if we don't now the specific type. Check [this post](https://stackoverflow.com/questions/51439843/unknown-vs-any) for more information.

Reply:
interesting post! We should shortly talk about that in the next meeting